host/xtest/pkcs11_1000.c

```diff
@@ -1463,3 +1483,492 @@ static void xtest_pkcs11_test_1007(ADBG_Case_t *c)
 }
 ADBG_CASE_DEFINE(pkcs11, 1007, xtest_pkcs11_test_1007,
 		"PKCS11: Check operations release at session closure");
+
+#define CK_MAC_KEY_HMAC(_type, _key_array) \
```

Comment:
I think this macro should also set `CKA_SIGN` and `CKA_VERIFY` attributes (to `CK_TRUE`) unless what these keys should not be allowed to be used for signing and/or verifying signature, according to the pkcs11 spec.

Reply:
I've found where this comes from. I will check and tell you my view.

Reply:
These are set by default to TRUE.

Reply:
Ok, thanks.

Reply:
... and many thanks the P-Rs.

Reply:
Added C_SIGN and C_VERIFY in the attribute array